src/city-council-district/city-council-district.service.ts

```diff
@@ -28,6 +32,32 @@ export class CityCouncilDistrictService {
     return await this.cityCouncilDistrictRepository.findTiles(params);
   }
 
+  async findGeoJsonById({
+    cityCouncilDistrictId,
+  }: FindCityCouncilDistrictGeoJsonByCityCouncilDistrictIdPathParams) {
+    const cityCouncilDistrictGeoJson =
+      await this.cityCouncilDistrictRepository.findGeoJsonById({
+        cityCouncilDistrictId,
+      });
+    if (cityCouncilDistrictGeoJson === undefined)
+      throw new ResourceNotFoundException();
+
+    const geometry = JSON.parse(cityCouncilDistrictGeoJson.geometry);
+    const properties = produce(
```

Comment:
Two things:
* I'm not sure this warrants adding immer as a dependency. This is API code so not concerned about bundle size, but concerned in terms of adding more code we depend on being maintained.
* When I pulled this branch `draft` shows up as a TS error because it is implicitly typed `any`. Is that happening for you? We should avoid that.

Maybe building `properties` manually isn't the worst thing for entities like this that don't have many properties? I don't think you would need this immer code or the `delete` if line 56 becomes
```
properties: { id: cityCouncilDistrictGeoJson.id},
```

Reply:
For point one, I understand that it looks silly when there is only one property. However, we will have several endpoints returning GeoJSON with various properties. The properties of each endpoint is also prone to evolve. For example, we are implementing a Capital Project GeoJson endpoint for #342. This endpoint has several properties already. It also reasonable to anticipate the exact properties changing over time.

Using immer to separate the properties from the geometry gives us a simple rule that implements a robust,  low maintenance, and general solution.

Reply:
For point two, VSCode language server across both my Linux and Mac machines recognize it as `Partial<CityCouncilDistrictEntity`. Also, `npm run typecheck` does not flag it as an error. 

Have you run `npm i` on the branch to install `immer`?